examples/binary/run.sh

```diff
@@ -0,0 +1,18 @@
+set -x
+set -e
+# start artipie
+docker run --name artipie -d -it -v $(pwd)/artipie.yaml:/etc/artipie.yml -v $(pwd):/var/artipie -p 8080:80 artipie/artipie:latest
+# wait for container to be ready for the new connections
+sleep 5
+# create a file for subsequent publication
+echo "hello world" > text.txt
+# publish text.txt
+curl --silent -X PUT --data-binary "@text.txt" http://localhost:8080/repo/text.txt
+# download the file
+STATUSCODE=$(curl --silent --output /dev/stderr --write-out "%{http_code}" -X GET http://localhost:8080/repo/text.txt)
+# remove container
+docker rm -f artipie
```

Comment:
@Sammers21 do not see why we need to remove running Artipie container in the example

Reply:
@olegmoz, because the second attempt to run the example will fail, a user will be tempted to do it by himself

Reply:
@Sammers21 I do not see that often in examples or documentations. Why do you think user will need to run it twice?